src/main/resources/db/changelog/changes/05-add-roles-to-db.yaml

```diff
@@ -8,10 +8,10 @@ databaseChangeLog:
             columns:
               - column:
                   name: name
-                  value: ROLE_ADMIN
+                  value: ADMIN
```

Comment:
I'm not sure whether it is a good idea to update existing Liquibase migrations

please be careful with that and read [this](https://www.liquibase.com/blog/dealing-with-changing-changesets)

I would suggest to create a new changeSet, where you would update the existing DB records

Reply:
I understand, just catching possibility to drop tables, while DB is empty, trying to not make my changelog a mess, before i even finish it =)